README.md

```diff
@@ -1,5 +1,10 @@
 
 this is a portfolio project that the purpose of this project is to understand how to create UIs adaptable to different screen sizes using media queries and understand how to parse a Figma design to create a UI.
+
+the site is published at :
+https://saeqasultani.github.io/Portfolio-Project/
+
+
 # Built With
```

Comment:
- [ ] [optional] It would be nice specifying the languages and frameworks used in the project, instead of just adding the titles.